website/project/metadata/real-world-evidence-2.json

```diff
@@ -0,0 +1,349 @@
+{
+	"atomicSchema": true,
+	"name": "Real World Evidence Recommended Minimum Study Registration Template",
+	"version": 3,
+	"description": null,
+	"blocks": [{
+		"help_text": "",
+		"block_type": "page-heading",
+		"display_text": "Administrative Information",
+		"example_text": "",
+		"required": false
+	}, {
+		"help_text": "The research question is essential to guide the research project and should pinpoint exactly what you are attempting to evaluate. For hypothesis evaluating treatment effect or comparative effectiveness research the question should include the population of interest, exposure, comparator and outcome(s) to be evaluated.",
+		"block_type": "question-label",
+		"display_text": "Research question",
+		"example_text": "Does in-hospital zxxxxxnib improve pulmonary function compared to yxxxxxnib in\npatients treated with high-dose corticosteroids for moderate to severe SARS-CoV-2\npneumonia?\nWhat are the comparative risks for stroke or major bleeding in patients with non-\nvalvular atrial fibrillation who initiate anticoagulant X versus anticoagulant Y?",
```

Comment:
No line-break before "patients" or "pneumonia" or "valvular". That's just sheets being weird. If you click on the cell, it re-formats correctly.